administrator/components/com_admin/sql/updates/postgresql/4.1.0-2021-06-08.sql

```diff
@@ -0,0 +1,26 @@
+--
+-- Table structure for table "#__cookiemanager_cookies"
+--
+
+CREATE TABLE IF NOT EXISTS "#__cookiemanager_cookies" (
+  "id" serial NOT NULL PRIMARY KEY,
+  "title" varchar(255) NOT NULL,
+  "alias" varchar(400) NOT NULL,
+  "cookie_name" varchar(255) NOT NULL,
+  "cookie_desc" varchar(255) NOT NULL,
+  "exp_period" varchar(20) NOT NULL,
+  "exp_value" integer DEFAULT 0 NOT NULL,
+  "catid" integer DEFAULT 0 NOT NULL,
+  "published" smallint DEFAULT 1 NOT NULL,
+  "ordering" integer DEFAULT 0 NOT NULL,
+  "created" timestamp without time zone NOT NULL,
+  "created_by" integer DEFAULT 0 NOT NULL,
+  "modified" timestamp without time zone NOT NULL,
+  "modified_by" integer DEFAULT 0 NOT NULL
```

Comment:
```suggestion
  "modified_by" integer DEFAULT 0 NOT NULL,
  PRIMARY KEY ("id")
```
Primary key has to be specified in this way.